configure.ac

```diff
@@ -88,6 +88,12 @@ AC_ARG_ENABLE([mining],
   [enable_mining=$enableval],
   [enable_mining=yes])
 
+AC_ARG_ENABLE([rust],
+  [AS_HELP_STRING([--enable-rust],
```

Comment:
Should be `--disable-rust` here and in the help text (but leaving `enable_rust` as-is), as the default is to enable. Upstream has fixed this for e.g. the wallet, and I should have done the same for `-mining` when I added it. Non-blocking, as `AC_ARG_ENABLE` [transparently defines](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Package-Options.html) `--disable-foo` as `--enable-foo=no`.